scripts/ColumnConsistency2015to2019Vs2024.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import requests
+from io import StringIO
+import time
+
```

Comment:
Can you provide a comment block that describes the various functions and logic blocks? Our developers will likely not easily have access to your ticket, and they will need a brief explainer of the motivation for this file.